NETProvider/src/FirebirdSql.Data.FirebirdClient/Client/Common/XsqldaMarshaler.cs

```diff
@@ -72,13 +50,14 @@ public void CleanUpNativeData(ref IntPtr pNativeData)
 					IntPtr ptr1 = GetIntPtr(pNativeData, ComputeLength(i));
 
 					// Free	sqldata	and	sqlind pointers	if needed
-					XSQLVAR sqlvar = (XSQLVAR)Marshal.PtrToStructure(ptr1, typeof(XSQLVAR));
+					XSQLVAREmpty sqlvar = (XSQLVAREmpty)Marshal.PtrToStructure(ptr1, typeof(XSQLVAREmpty));
```

Comment:
Ahh, I see.

Just out of curiosity, what if there will be another `IntPtr` at the end in the future. Then the "empty" approach would not work, right?